pyproject.toml

```diff
@@ -2,7 +2,7 @@
 requires = [
     "setuptools>=42",
     "wheel",
-    "scikit-build==0.14.0",
+    "scikit-build",
```

Comment:
It is a good practice to specify a version for `scikit-build` dependencies for consistent build. On a second thought, I should probably migrate from `scikit-build` to `scikit-build-core`.